src/mmrelay/cli.py

```
@@ -180,6 +252,52 @@ def parse_arguments() -> argparse.Namespace:
     # Add grouped subcommands for modern CLI interface
     subparsers = parser.add_subparsers(dest="command", help="Available commands")
 
+    # PATHS command (top-level)
+    paths_parser = subparsers.add_parser(
```

Ruff (0.14.14)

[error] Local variable `paths_parser` is assigned to but never used

Remove assignment to unused variable `paths_parser`

(F841)